mitodl/mit-open
# Fix missing style bundle

#### What are the relevant tickets?
None

#### What's this PR do?
Fixes a webpack loader error and a cookiecutter template rendering bug

#### How should this be manually tested?
Site should load. You should see a rounded blue rectangle and a checkbox which you should be able to click and cause a change of state
